src/api.v2/helpers/modelGuts.js

```diff
@@ -0,0 +1,66 @@
+const sqlClient = require('../../sql/sqlClient');
```

Comment:
this file is so beautiful 😍 goodbye, hacky dynamodb inserts, custom to each field and table!